core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3403,9 +3403,6 @@ public void onAttained(Milestone milestone) {
                     LOGGER.log(lv, s);
                 }
             });
-        } catch (InterruptedException | ReactorException | IOException e) {
```

Comment:
Why is this removed?

Reply:
The short answer is "because the computer did it". A longer answer is that those exceptions don't apply there.

Reply:
I guess it's fine since they are not runtime ones.